src/app/Http/Controllers/Event/Store.php

```diff
@@ -4,24 +4,16 @@
 
 use Illuminate\Routing\Controller;
 use LaravelEnso\Calendar\app\Models\Event;
-use LaravelEnso\Calendar\app\Http\Requests\ValidateEventStore;
 use LaravelEnso\Calendar\app\Http\Resources\Event as Resource;
+use LaravelEnso\Calendar\app\Http\Requests\ValidateEventRequest;
 
 class Store extends Controller
 {
-    public function __invoke(ValidateEventStore $request, Event $event)
+    public function __invoke(ValidateEventRequest $request, Event $event)
```

Comment:
when creating an Event we want to make sure the user has access to that calendar.
We can either use a policy in the controller or make use of the `authorize()` method in the request validation class